### ISSUE TITLE
[ISSUE]  databricks_sql_endpoint doesn't change sizes

### ISSUE THREAD

**Issue by rtyler** [databrickslabs/terraform-provider-databricks#682]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Databricks Provider issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version

This is Terraform 0.13.1

### Affected Resource(s)

- `databricks_sql_endpoint`

![tf](https://user-images.githubusercontent.com/26594/121100226-3f968c80-c7ae-11eb-86d2-fd98ec66a08a.png)

Basically this diff didn't result in any planned or applied changes. Womp womp

**Comment by rtyler** [comment]
Oh bollocks, this does apply properly, found a different bug
